help/en/package/jmri/jmrit/etcs/index.shtml

```diff
@@ -0,0 +1,127 @@
+  <meta name="keywords" content="JMRI ERTMS ETCS DMI">
+  <!--#include virtual="/help/en/parts/Style.shtml" -->
+</head>
+<body>
+  <!--#include virtual="/help/en/parts/Header.shtml" -->
+
+  <div id="mBody">
+    <div id="mainContent" class="no-sidebar">
+      <h1>JMRI : ERTMS ETCS</h1>
+
+        <p>The European Rail Traffic Management System ( ERTMS ) aims to improve
+        interobability between the multiple systems in use by European railways,
+        making it easier for trains to cross national borders without needing to change systems.</p>
+
+        <p>European Train Control System ( ETCS ) is the signalling component of ERTMS.</p>
+
+
+      <h2>Driver Machine Interface Simulation</h2>
+
+      <span class="since">since 5.7.4</span>
+      <p>
+      The Driver Machine Interface  ( DMI, the in-cab screen which displays speed and signalling
+      information to the driver ) is one of the most recognisable elements of ETCS.
+      </p>
+
+      <a href="etcs_did_640x478.png">
+      <img src="etcs_did_640x478.png" alt="ERTMS ETCS JMRI DMI" width="320" height="239"></a>
+
+      <p>The DMI can include information about speed, track conditions, and system alerts.</p>
+
+      <p>It is possible to simulate DMI screens and control their content programmatically,
+      currently there is no train control logic within the DMI Panel.</p>
+
+      <p>The DMI follows the touch screen layout ( as apposed to the soft key layout ),
+      as it is able to receive button and mouse clicks directly.</p>
+
+      <p>The JMRI Fast Clock is used as the source for the time display.</p>
+
+      <h3>Launching DMI Displays with Jython</h3>
+
+      <p>DMI Displays can be launched via <a href="https://www.jmri.org/help/en/html/tools/scripting/index.shtml">JMRI Jython Scripting</a>.</p>
+
+
+<div style="margin:10px; padding:20px;">
+<code>import jmri<br>
+from jmri.jmrit.etcs.dmi.swing import DmiFrame<br>
+<br>
+f = DmiFrame()<br>
+f.setVisible(True)<br>
+<br>
+dmiPanel = f.getDmiPanel()<br>
+dmiPanel.setActualSpeed(100)<br>
+<br>
+</code>
+</div>
+
+      <p>This screen is now viewable on other devices via the JMRI Web Server.</p>
+
+      <p>For the full list of available methods, see the <a href=
+      "https://www.jmri.org/JavaDoc/doc/jmri/jmrit/etcs/dmi/swing/DmiPanel.html">DmiPanel JavaDoc</a>.</p>
+      <p>The <a href=
+      "https://github.com/JMRI/JMRI/blob/master/java/src/jmri/jmrit/etcs/dmi/swing/DmiDemo.java"
+      >code for the DMI Demonstration</a> also provides examples of how to manipulate
+      the panel.</p>
+
+
+
+
+    <h3>Installing the Graphics</h3>
+      <p>JMRI does not yet contain all of the graphics required to display the full DMI, however
+      the European Rail Association has example ETCS graphics for download : </p>
+        <ul>
+        <li><a href="https://www.era.europa.eu/era-folder/1-ccs-tsi-appendix-mandatory-specifications-etcs-b4-r1-rmr-gsm-r-b1-mr1-frmcs-b0-ato-b1">ETCS 4</a></li>
```

Comment:
Is this download step needed because of e.g. copyright?  We could certainly include these files in the JMRI distribution. If we omit the PDF file, they're not all that large.

Reply:
They've got a permissive copyright statement at https://www.era.europa.eu/content/disclaimer-and-copyright-notice   I think it would be sufficient to have a README.md file in the relevant directories that points back to the source location and the copyright page.  

Because of the size of that, let's merge this and then come back with another PR for the images, if you'd like to do that.

Reply:
Sounds good, I'll prepare a follow-up PR with the images :-)